lib/platform/gitlab/index.ts

```diff
@@ -432,18 +432,25 @@ async function tryPrAutomerge(
   if (platformOptions?.gitLabAutomerge) {
     try {
       const desiredStatus = 'can_be_merged';
-      const retryTimes = 5;
+      const cannotBeMerged = `cannot_be_merged`;
+      const retryTimes = 100;
```

Comment:
```suggestion
      const retryTimes = 5;
```

No more that 5 tries, increase delay instead.

Reply:
15 seconds is preferable for you?

Reply:
I think we should not retry more that 5 times, but instead increase the delay computation.